apps/partners/src/components/ProfilePopover.tsx

```
@@ -0,0 +1,100 @@
+        </div>
+        <div className="max-h-60 overflow-y-auto">
+          {organizations.map((org) => (
+            <button
+              key={org.id}
+              className={`w-full text-left px-4 py-2 flex items-center gap-2 hover:bg-accent hover:text-accent-foreground ${currentOrg?.id === org.id ? "bg-gray-100 font-bold" : ""}`}
+              onClick={() => {
+                setCurrentOrg(org)
+                setOpen(false)
+                router.push(`/organization/${org.id}`)
+              }}
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)